hack/testing/test-010-deploy-via-olm-minimal.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# Given an OLM manifest, verify a green field deployment
+# of cluster logging by asserting CLO creates the resources
+# that begets the operands that make up logging.
+
+set -e
+if [ "${DEBUG:-}" = "true" ]; then
+	set -x
+fi
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../lib/init.sh"
+source "$(dirname $0)/assertions"
+
+os::test::junit::declare_suite_start "${BASH_SOURCE[0]}"
+
+ARTIFACT_DIR=${ARTIFACT_DIR:-"$(pwd)/_output"}
+if [ ! -d $ARTIFACT_DIR ] ; then
+  mkdir -p $ARTIFACT_DIR
+fi
+export NAMESPACE="openshift-logging"
+repo_dir="$(dirname $0)/../.."
+manifest=${repo_dir}/manifests
+version=$(basename $(find $manifest -type d | sort -r | head -n 1))
+
+cleanup(){
+  local return_code="$?"
+  set +e
+  oc logs -n ${NAMESPACE} deployment/cluster-logging-operator > $ARTIFACT_DIR/cluster-logging-operator.log 2>&1 ||:
+  oc delete ns ${NAMESPACE} --wait=false --ignore-not-found
+  oc delete crd elasticsearches.logging.openshift.io --wait=false --ignore-not-found
+
+  os::cleanup::all "${return_code}"
+  
+  exit ${return_code}
+}
+trap cleanup exit
+
+if [ -n "${IMAGE_CLUSTER_LOGGING_OPERATOR:-}" ] ; then
+  source "$(dirname $0)/../common"
+fi
+if [ -n "${IMAGE_FORMAT:-}" ] ; then
+  IMAGE_CLUSTER_LOGGING_OPERATOR=$(sed -e "s,\${component},cluster-logging-operator," <(echo $IMAGE_FORMAT))
+else
+  IMAGE_CLUSTER_LOGGING_OPERATOR=${IMAGE_CLUSTER_LOGGING_OPERATOR:-quay.io/openshift/origin-cluster-logging-operator:latest}
+fi
+
+KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
+
+oc create ns ${NAMESPACE} || :
+
+
+os::cmd::expect_success "oc create -f ${repo_dir}/vendor/github.com/openshift/elasticsearch-operator/manifests/${version}/elasticsearches.crd.yaml"
+
+
+os::log::info "Deploying operator from ${manifest}"
+NAMESPACE=${NAMESPACE} \
+VERSION=${version} \
+OPERATOR_IMAGE=${IMAGE_CLUSTER_LOGGING_OPERATOR} \
```

Comment:
what about the operand image name?    ideally that should also come from the IMAGE_FORMAT so we can get the one built by the PR and reuse this in the aggregated logging repo

Reply:
This particular test does no verification specific to the deployed operand; it is basically a revised version of the original test which only validated the operator produced the desired resources.  I would envison that we either add more tests or expand this one to possibly validate the image used by the operand

Reply:
but i assume someday you'll want to invoke this test from your aggregated-logging repo also.  So is there any reason not to make it fully parameterized to consume "the images built by the PR job" in preparation for that?